pom.xml

```diff
@@ -374,9 +374,6 @@
     <rome.version>1.18.0</rome.version>
     <rhino.version>1.7.13</rhino.version>
     <snakeyaml.version>1.33</snakeyaml.version>
-    <sqlite-jdbc.version>3.41.2.2</sqlite-jdbc.version>
```

Comment:
Are you sure you want to delete these 3? This was just put in 4 days ago. Maybe it's a merge error?

https://github.com/pentaho/maven-parent-poms/blame/master/pom.xml#L377-L379

Reply:
You are right, let me fix that